src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.machines.Bulldozer;
+import core.basesyntax.machines.Excavator;
+import core.basesyntax.machines.Machine;
+import core.basesyntax.machines.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+
```

Comment:
Redundant empty line. [See here ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#please-dont-add-redundant-empty-lines-to-your-code)